_data/site.js

```diff
@@ -17,6 +17,11 @@ function calculateUrl() {
 module.exports = {
     title: "ESLint - Pluggable JavaScript linter",
     description: "A pluggable and configurable linter tool for identifying and reporting on patterns in JavaScript. Maintain your code quality with ease.",
+    banner: {
+        text: null,
```

Comment:
This won't render anything at the moment. I can update this with copy either in this PR or in a follow up once we finalize it.

Reply:
Nice.

Reply:
Disclaimer: my understanding of the template syntax is limited. Would we be able to include links within this string? That’s the only improvement I could see wanting if that doesn’t work currently.

Reply:
I wasn’t sure either, but I verified this does work!